tokio-util/src/io/sync_bridge.rs

```diff
@@ -151,7 +150,7 @@ use tokio::io::{
 ///     // Example: In-memory data.
 ///     let data = b"Hello, world!"; // A byte slice.
 ///     let reader = Cursor::new(data); // Create an in-memory AsyncRead.
-///     compress_data(reader).await;
+///     let _ = compress_data(reader).await;
```

Comment:
Let's not ignore errors.
```suggestion
///     compress_data(reader).await?;
```